go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/hashicorp/go-retryablehttp v0.7.1
+  github.com/jarcoal/httpmock-1.2.0
```

Comment:
we have the same library version defined in line 9

and that's actually not valid format for go mod file, when I run `go mod tidy`:

```
go: errors parsing go.mod:
terraform-provider-twingate/go.mod:8: usage: require module/path v1.2.3
```